doc/source/getting_started/install.rst

```diff
@@ -270,6 +270,19 @@ For example, :func:`pandas.read_hdf` requires the ``pytables`` package, while
 optional dependency is not installed, pandas will raise an ``ImportError`` when
 the method requiring that dependency is called.
 
+Timezones
+^^^^^^^^^
+
+========================= ========================= =============================================================
+Dependency                Minimum Version           Notes
+========================= ========================= =============================================================
+tzdata                    2022.1(pypi)/             Allows the use of ``zoneinfo`` timezones with pandas.
+                          2022a(for system tzdata)  **Note**: You only need to install the pypi package, if your
```

Comment:
Looks like the `2022a` is a conda forge package? 
https://anaconda.org/conda-forge/tzdata
https://github.com/eggert/tz

Which is a different project from tzdata (`2022.1`).
https://pypi.org/project/tzdata/#history
https://github.com/python/tzdata

I'm not sure it's a good idea to recommend 2 differently maintained packages here. Can we just recommend the pypi one since it's the once recommended in the official docs https://docs.python.org/3/library/zoneinfo.html#data-sources?

Reply:
I think we'd still want to state a min version for system tzdata for consistency. 2022a is also the version of the actual IANA tz db(it's python's tzdata versioning scheme that's different).

Reply:
Ah okay sounds reasonable then. Might be good then to mention that the system tzdata can be updated by installing the conda forge package?